src/plugins/discover/public/application/_discover.scss

```
@@ -103,13 +103,11 @@ discover-app {
   left: -10000px;
   top: $euiSizeXS;
```

Then this would simply become.

```suggestion
  right: $euiSizeXS;
  top: $euiSizeXS;
```

And you can delete everything below this.